lib/bitvec.cpp

```diff
@@ -137,3 +137,12 @@ unsigned bitvec::ffz(unsigned start) const {
 #endif
     return rv;
 }
+
+bool bitvec::is_contiguous() const {
+    // Empty bitvec is not contiguous
+    if (popcount() == 0)
```

Comment:
using `if (empty()) return false;` might be clearer?  I guess if you're using popcount() anyways.